src/core/plugins/configs/actions.js

```diff
@@ -19,8 +24,35 @@ export function toggle(configName) {
   }
 }
 
-
 // Hook
 export const loaded = () => () => {
```

Comment:
Removing this and its usage https://github.com/swagger-api/swagger-ui/blob/543f0882db836bf40eb79b8bfa4aa1c0bdab5698/src/core/index.js#L75
breaks deep linking - we get the correct URL but if we refresh/go to this URL, the site doesn't scroll to the operation and it's not expanded.